scripts/Muon/GUI/FrequencyDomainAnalysis/FFT/fft_model.py

```diff
@@ -122,13 +122,14 @@ def FFTAlg(self, FFTInputs):
         for name, value in iteritems(FFTInputs):
             self.alg.setProperty(name, value)
         self.alg.execute()
+        ws = self.alg.getProperty("OutputWorkspace").value
         mantid.AnalysisDataService.addOrReplace(
             FFTInputs["OutputWorkspace"],
             self.alg.getProperty("OutputWorkspace").value)
 
-        ws = self.alg.getPropertyValue("OutputWorkspace")
+        ws.getAxis(0).setUnit('Label').setLabel('Frequency', 'Hz')
```

Comment:
Could we hold off on mergeing I think this might MHz instead and just want to double check.

Reply:
@Mantid-Matthew Have you had a chance to check this out?